srcpkgs/XyGrib/template

```diff
@@ -13,6 +14,8 @@ homepage="https://opengribs.org/"
 distfiles="https://github.com/opengribs/XyGrib/archive/v${version}.tar.gz"
 checksum=2dc6099293ae6f7a4bfbfc0cab590cf7ad48241d608e6d7a76e35961b9fc2157
 
+export CMAKE_GENERATOR="Unix Makefiles"
```

Comment:
does it not work with ninja?

Reply:
No, builds failed with `unknown target`.

Reply:
I just set up a new build env and checked again, and build now does work with ninja. Will update branch when Python 3.11 settles.

Reply:
I could reproduce this "unknown target" in attempt to crosscompile because of this block:

```diff
if [ "$CROSS_BUILD" ]; then
	make_build_args="LINK=${XBPS_CROSS_TRIPLET}-c++"
fi
```

Removal of that block did not bring up any other issues.

Reply:
Thanks, confirmed.